docs/Actions.md

```diff
@@ -349,11 +349,22 @@ s3(
   file: 'AppName.ipa',                            # This would come from IpaAction.
   dsym: 'AppName.app.dSYM.zip',                   # This would come from IpaAction.
   path: 'v{CFBundleShortVersionString}_b{CFBundleVersion}/' # This is actually the default.
+  version_file_name,                              # Name of the file to upload to S3. Defaults to 'version.son'
```

Comment:
That doesn't look like a valid configuration, have you tested your feature locally?